### PR TITLE
B/bulk bugfixes

### PR DESCRIPTION
**Pull request by selankon**
Multiple bug fixing

### CODE REVIEW

File: components/pages/app/footer.tsx [hunk 1]
```diff
@@ -1,11 +1,59 @@
...
+  const { i18n } = useTranslation();
+
+  const LINKS: HeaderLink[] = [
+    // {
+    //   url: PRIVACY_PATH,
+    //   name: i18n.t("links.privacy_policy"),
+    //   external: true,
+    //   logged: true,
+    //   guest: true
+    // },
```
Comment (elboletaire) [inline components/pages/app/footer.tsx:18]: Avoid leaving commented code please

Comment (emmdim) [reply components/pages/app/footer.tsx:18]: fyi @selankon  :point_up:

File: components/pages/app/footer.tsx [hunk 2]
```diff
@@ -1,11 +1,59 @@
...
+    {
+      url: 'https://blog.vocdoni.io',
+      name: i18n.t('links.blog'),
+      external: true,
+      logged: true,
+      guest: true,
+    },
+    {
+      url: 'https://docs.vocdoni.io',
```
Comment (emmdim) [inline components/pages/app/footer.tsx:27]: I think this should point to `https://developer.vocdoni.io` if I am not mistaken.
cc @jpaulet

File: components/pages/app/footer.tsx [hunk 3]
```diff
@@ -1,11 +1,59 @@
...
+      name: i18n.t('links.docs'),
+      external: true,
+      logged: true,
+      guest: true,
+    },
+    {
+      url: 'https://help.aragon.org/collection/54-vocdoni-user-guide',
```
Comment (emmdim) [inline components/pages/app/footer.tsx:34]: This link does not work correctly. Where should it point to? cc @jpaulet

File: components/pages/app/footer.tsx [hunk 4]
```diff
@@ -1,11 +1,59 @@
...
+  const LINKS: HeaderLink[] = [
+    // {
+    //   url: PRIVACY_PATH,
+    //   name: i18n.t("links.privacy_policy"),
+    //   external: true,
+    //   logged: true,
+    //   guest: true
+    // },
+    {
+      url: 'https://blog.vocdoni.io',
```
Comment (emmdim) [inline components/pages/app/footer.tsx:20]: FYI this leads in the aragon based blog @jpaulet 
Is this ok?

File: i18n/locales/ca.json [hunk 5]
```diff
@@ -339,7 +339,8 @@
     "vote": {
         "badge": {
             "anonymous": "Anònim",
-            "archived": "Arxivat"
+            "archived": "Arxivat",
+            "invalid": "Invalida"
```
Comment (elboletaire) [inline i18n/locales/ca.json:343]: Should be `Invàlida` (missing accent)

File: i18n/locales/es.json [hunk 6]
```diff
@@ -339,7 +339,8 @@
     "vote": {
         "badge": {
             "anonymous": "Anónimo",
-            "archived": "Archivada"
+            "archived": "Archivada",
+            "invalid": "Invalida"
```
Comment (elboletaire) [inline i18n/locales/es.json:343]: Should be "Inválida" (missing accent)

**Review by elboletaire — CHANGES_REQUESTED** [review]

**Review by emmdim — COMMENTED** [review]
There are a series of warning during the build.
Could please you eliminate the ones related to 
`@typescript-eslint/no-unused-vars`
and `react-hooks/exhaustive-deps` @selankon ?
You can find them by doing a build locally as well

File: components/pages/app/footer.tsx (hunk 2 above)
Comment (jpaulet) [reply components/pages/app/footer.tsx:27]: 100%, it must point to https://developer.vocdoni.io

File: components/pages/app/footer.tsx (hunk 3 above)
Comment (jpaulet) [reply components/pages/app/footer.tsx:34]: Right now, I would point this link to our "Support" channel on Discord: https://discord.gg/sM7UhAGY53

File: components/pages/app/footer.tsx (hunk 4 above)
Comment (jpaulet) [reply components/pages/app/footer.tsx:20]: For the moment yes. We are moving the blog from Aragon->Vocdoni, but the work will start this week/next week. It's OK for now to point to the Aragon blog, but **changing the current link to: https://blog.aragon.org/vocdoni/**

File: components/pages/app/footer.tsx (hunk 3 above)
Comment (selankon) [reply components/pages/app/footer.tsx:34]: FYI @jpaulet there are already a discord channel on the "support" link:

https://github.com/vocdoni/explorer-ui/blob/8485488e8d23027fcf373c6eab89885aec327bd9/components/pages/app/footer.tsx#L41-L47

Which is invalid. Should I preserve the help link or the support one? (correcting with the working discord link)

File: components/pages/app/footer.tsx (hunk 4 above)
Comment (selankon) [reply components/pages/app/footer.tsx]: beb440e

File: i18n/locales/es.json (hunk 6 above)
Comment (selankon) [reply i18n/locales/es.json]: beb440e

File: i18n/locales/ca.json (hunk 5 above)
Comment (selankon) [reply i18n/locales/ca.json]: beb440e

File: components/pages/app/footer.tsx (hunk 1 above)
Comment (selankon) [reply components/pages/app/footer.tsx]: beb440e

File: components/pages/app/footer.tsx (hunk 3 above)
Comment (jpaulet) [reply components/pages/app/footer.tsx]: Then, we can remove the "help" link and update the support one with the new link.

**Comment by selankon** [comment]
> There are a series of warning during the build. Could please you eliminate the ones related to `@typescript-eslint/no-unused-vars` and `react-hooks/exhaustive-deps` @selankon ? You can find them by doing a build locally as well

This is done on 87b5eb5

**Comment by selankon** [comment]
This is ready for merge @emmdim

**Review by elboletaire — APPROVED** [review]

**Pull request merged**